articles/container-apps/azure-arc-create-container-app.md

```diff
@@ -112,7 +114,7 @@ let StartTime = ago(72h);
 let EndTime = now();
 ContainerAppsConsoleLogs_CL
 | where TimeGenerated between (StartTime .. EndTime)
-| where AppName_s =~ "myContainerApp"
+| where AppName_s =~ "my-container-app"
```

Comment:
Shouldn't this now be ...
```suggestion
| where AppName_s =~ $myContainerApp
```

Reply:
This is executed in the log analysis workspace, not in bash, so need to enter the actual value directly.

Reply:
Hi, I abandon this pr, my colleague merged my changes in this pr: https://github.com/MicrosoftDocs/azure-docs/pull/102998
Many thanks for your review!